### DISCUSSION TITLE
Has the project been closed?

### DISCUSSION THREAD

**Discussion by joseciceroassisjr** [HandyGram/app#8] [General]
I found the file these days and it works great on wear Os but I would like to know if there will be future updates

**Comment by johwconst** [comment]
Everything indicates that the project is not at a standstill. The developer is posting project updates on his Telegram channel: https://t.me/handygram_client

For development logs go to: https://t.me/handygram_devlog

**Comment by tdrkDev** [comment]
Project isn't dead.
